### PR TITLE
thermo couples fully working

### PR DESCRIPTION
**Pull request by fazerlicourice7**

### CODE REVIEW

File: libraries/thermocouple/Thermocouple.cpp [hunk 1]
```diff
@@ -3,26 +3,37 @@
...
-  if(whichSensor >= numSensors){
+  if(whichSensor >= numThermos) {
     data[0] = -1;
     return;
   }
 
   scrollToRightSensor(whichSensor);
 
-  for (int i = 0; i < 9; i++){ // need 9 bytes apparently.
+  for (int i = 0; i < 9; i++) { // need 9 bytes apparently.
```
Comment (vainaviv) [inline libraries/thermocouple/Thermocouple.cpp:36]: I see it's reading 9 bytes but each float is only 4 bytes in the way I've implemented the I2C communication. Just wanted to make sure this won't be a problem.

Comment (fazerlicourice7) [reply libraries/thermocouple/Thermocouple.cpp:36]: That's fine. See the lines below, not all of that data is the temperature reading.
Some of it is other metadata

**Review by vainaviv — APPROVED** [review]
Looks great!

**Comment by fazerlicourice7** [comment]
bruh. you just pointed out things I need to add (deactivate high pressure, move the toggles back to private, etc). Don't approve!

**Review by vainaviv — CHANGES_REQUESTED** [review]
Add a way to deactivate high pressure solenoid.

**Review by vainaviv — CHANGES_REQUESTED** [review]
I still need a way to control LOX Gems and Propane Gems.

**Review by vainaviv — APPROVED** [review]
Looks great!

**Pull request merged**